config/redirects

```diff
@@ -212,10 +212,10 @@ raw: docs/drivers/drivers/go/ -> ${base}/go/current/
 raw: docs/drivers/drivers/java/ -> ${base}/java-drivers/
 raw: docs/drivers/drivers/kotlin/ -> ${base}/kotlin-drivers/
 raw: docs/drivers/drivers/node/ -> $${base}/node/current/
-raw: docs/drivers/drivers/php/ -> ${base}/php/
-raw: docs/drivers/drivers/python/ -> ${base}/python/
+raw: docs/drivers/drivers/php/ -> ${base}/php-drivers/
+raw: docs/drivers/drivers/python/ -> ${base}/python-drivers/
```

Comment:
@ccho-mongodb These redirects don't seem to be working. In staging, I still see the old page, which now no longer shows the correct navbar entries. Do these redirects only take effect after deployment?

Reply:
Redirects aren't written to the staging environment, only the production environment. Usually to build confidence that they will work in production, we generate the redirects output with mut-redirects, and paste that into the PR. Then, we test them in production once deployed.